packages/@o3r/pipeline/testing/jest.config.ut.builders.js

```diff
@@ -1,5 +1,6 @@
 const path = require('node:path');
 const getJestProjectConfig = require('../../../../jest.config.ut').getJestProjectConfig;
+
```

Comment:
why the extra line?

Reply:
we should have an empty line after imports